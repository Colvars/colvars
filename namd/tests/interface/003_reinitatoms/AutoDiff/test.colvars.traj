# step       
           0 
# step         rgyr                  ft_rgyr               fa_rgyr               
           0    6.87962472939909e+00  0.00000000000000e+00 -8.79624729399095e-01 
           1    6.87884690576167e+00  9.41011309459353e+01 -8.78846905761671e-01 
           2    6.87828544620917e+00  9.04041844059973e+01 -8.78285446209173e-01 
           3    6.87791917633219e+00  8.14481165806774e+01 -8.77919176332190e-01 
           4    6.87771596366921e+00  6.78179862575362e+01 -8.77715963669207e-01 
           5    6.87763583984369e+00  5.08218206455768e+01 -8.77635839843695e-01 
           6    6.87763510984747e+00  3.21944582169423e+01 -8.77635109847473e-01 
           7    6.87767070515395e+00  1.37842105362567e+01 -8.77670705153955e-01 
           8    6.87770424862566e+00 -2.67218755378605e+00 -8.77704248625659e-01 
           9    6.87770538491566e+00 -1.57406166805562e+01 -8.77705384915658e-01 
          10    6.87765396033178e+00 -2.44703048777650e+01 -8.77653960331782e-01 
          11    6.87754086425991e+00 -2.84781933933915e+01 -8.77540864259909e-01 
          12    6.87736785259684e+00 -2.78153473089100e+01 -8.77367852596840e-01 
          13    6.87714714763978e+00 -2.26302115807297e+01 -8.77147147639783e-01 
          14    6.87690017590140e+00 -1.34489740538511e+01 -8.76900175901402e-01 
          15    6.87665557061129e+00 -1.12489580901133e+00 -8.76655570611289e-01 
          16    6.87644626424549e+00  1.30850351689627e+01 -8.76446264245494e-01 
          17    6.87630623826635e+00  2.77657826415634e+01 -8.76306238266349e-01 
          18    6.87626719198701e+00  4.14611442641179e+01 -8.76267191987006e-01 
          19    6.87635569101452e+00  5.29205207769169e+01 -8.76355691014518e-01 
          20    6.87659105493692e+00  6.12164358381386e+01 -8.76591054936918e-01 
          21    6.87698392333700e+00  6.57238534788376e+01 -8.76983923337002e-01 
          22    6.87753533417408e+00  6.60523740356144e+01 -8.77535334174082e-01 
          23    6.87823626800109e+00  6.20259880441399e+01 -8.78236268001092e-01 
          24    6.87906740785677e+00  5.35705702753062e+01 -8.79067407856772e-01 
          25    6.87999931786412e+00  4.07931207935066e+01 -8.79999317864117e-01 
          26    6.88099335539587e+00  2.41083512952447e+01 -8.80993355395873e-01 
          27    6.88200354666071e+00  4.33097605073502e+00 -8.82003546660711e-01 
          28    6.88297948174718e+00 -1.73022028410802e+01 -8.82979481747185e-01 
          29    6.88387003807166e+00 -3.92114564583877e+01 -8.83870038071655e-01 
          30    6.88462761347109e+00 -5.96044127873818e+01 -8.84627613471091e-01 
          31    6.88521232913584e+00 -7.67001581966352e+01 -8.85212329135837e-01 
          32    6.88559571846137e+00 -8.89304384672354e+01 -8.85595718461371e-01 
          33    6.88576306814155e+00 -9.52939130057364e+01 -8.85763068141552e-01 
          34    6.88571427329494e+00 -9.54143821779247e+01 -8.85714273294942e-01 
          35    6.88546296350663e+00 -8.96484651775477e+01 -8.85462963506635e-01 
          36    6.88503394462706e+00 -7.90730655748384e+01 -8.85033944627065e-01 
          37    6.88445915091932e+00 -6.54097426609426e+01 -8.84459150919319e-01 
          38    6.88377274460901e+00 -5.07571883087193e+01 -8.83772744609010e-01 
          39    6.88300617631749e+00 -3.72442767426713e+01 -8.83006176317490e-01 
          40    6.88218411453200e+00 -2.66371930519242e+01 -8.82184114532001e-01 
# step         rgyr                  ft_rgyr               fa_rgyr               
          40    6.79302548471894e+00                  -nan -7.93025484718935e-01 
          41    6.79123478804224e+00  1.47884739101504e+01 -7.91234788042243e-01 
          42    6.78960135015216e+00  6.31388772773012e+01 -7.89601350152160e-01 
          43    6.78822556441034e+00  1.06473629117400e+02 -7.88225564410344e-01 
          44    6.78718836728478e+00  1.41615537771249e+02 -7.87188367284780e-01 
          45    6.78654524635830e+00  1.65951779056351e+02 -7.86545246358304e-01 
          46    6.78632262212756e+00  1.77868414642074e+02 -7.86322622127558e-01 
          47    6.78651664924570e+00  1.76780555403037e+02 -7.86516649245702e-01 
          48    6.78709447322938e+00  1.63158774650263e+02 -7.87094473229380e-01 
          49    6.78799770245370e+00  1.38435193019193e+02 -7.87997702453697e-01 
          50    6.78914772242091e+00  1.04850106085365e+02 -7.89147722420910e-01 
          51    6.79045239785112e+00  6.52615921470907e+01 -7.90452397851118e-01 
          52    6.79181359951293e+00  2.29066423056194e+01 -7.91813599512934e-01 
          53    6.79313491716379e+00 -1.88723942265534e+01 -7.93134917163787e-01 
          54    6.79432888384693e+00 -5.69200801246878e+01 -7.94328883846926e-01 
          55    6.79532313655605e+00 -8.85212915265862e+01 -7.95323136556054e-01 
          56    6.79606495915343e+00 -1.11649846148960e+02 -7.96064959153427e-01 
          57    6.79652397458270e+00 -1.25081475905039e+02 -7.96523974582702e-01 
          58    6.79669290881434e+00 -1.28439758480334e+02 -7.96692908814338e-01 
          59    6.79658645171890e+00 -1.22194713349709e+02 -7.96586451718904e-01 
          60    6.79623847172641e+00 -1.07555765691658e+02 -7.96238471726409e-01 
          61    6.79569795431237e+00 -8.63086903908918e+01 -7.95697954312366e-01 
          62    6.79502413487178e+00 -6.06025541671448e+01 -7.95024134871783e-01 
          63    6.79428098107406e+00 -3.28681758548632e+01 -7.94280981074062e-01 
          64    6.79353161313589e+00 -5.54971884901815e+00 -7.93531613135890e-01 
          65    6.79283311513736e+00  1.91013029508461e+01 -7.92833115137357e-01 
          66    6.79223198689355e+00  3.91369261893949e+01 -7.92231986893553e-01 
          67    6.79176060647312e+00  5.30671344131904e+01 -7.91760606473118e-01 
          68    6.79143508614340e+00  6.00127592666783e+01 -7.91435086143399e-01 
          69    6.79125457657879e+00  5.97131527409523e+01 -7.91254576578791e-01 
          70    6.79120252442337e+00  5.27280325076440e+01 -7.91202524423368e-01 
          71    6.79124957766182e+00  4.02961819632216e+01 -7.91249577661819e-01 
          72    6.79135771208333e+00  2.41520353313739e+01 -7.91357712083332e-01 
          73    6.79148519437635e+00  6.37107444494932e+00 -7.91485194376348e-01 
          74    6.79159174552850e+00 -1.08855284267396e+01 -7.91591745528497e-01 
          75    6.79164302206790e+00 -2.57303365775670e+01 -7.91643022067904e-01 
          76    6.79161384887261e+00 -3.67778734559073e+01 -7.91613848872609e-01 
          77    6.79149044924553e+00 -4.30336775225266e+01 -7.91490449245526e-01 
          78    6.79127037909875e+00 -4.44583315428767e+01 -7.91270379098752e-01 
          79    6.79096084019922e+00 -4.16949366512701e+01 -7.90960840199217e-01 
          80    6.79057632779113e+00 -3.56822005347099e+01 -7.90576327791133e-01 
