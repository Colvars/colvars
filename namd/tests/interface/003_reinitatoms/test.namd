# -*- tcl -*-

set mol_name "15ala2x"
set pbc "no"
set coor_pdb_file "../Common/${mol_name}.pdb"
set coor_bin_file "../Common/${mol_name}.init.coor"
set vel_bin_file "../Common/${mol_name}.init.vel"
set xsc_file "../Common/${mol_name}.init.xsc"
set par_files [list "../Common/par_all36_prot.prm"]

proc num_atoms {} {
    if { ${mol_name} == "15ala2x" } {
        return 324
    }
    if { ${mol_name} == "15leu2x" } {
        return 594
    }
    return -1
}

source ../Common/delete_tmp_files.tcl

source ../Common/common.namd

colvars on

run 0

# Need a longer run than stepspercycle to avoid GlobalMaster bugs
set numsteps [expr 2*[stepspercycle]]

foreach mol_name [list "15ala2x" "15leu2x"] {
    structure "../Common/${mol_name}.psf" pdb "../Common/${mol_name}.pdb"
    reinitatoms "../Common/${mol_name}.init"
    cv reset
    cv config "colvarsTrajFrequency 1"
    cv configfile test.in
    run ${numsteps}
}

source ../Common/delete_tmp_files.tcl
