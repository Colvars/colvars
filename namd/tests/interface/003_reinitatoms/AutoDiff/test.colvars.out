colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: Opening trajectory file "test.colvars.traj".
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Resetting the Collective Variables module.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration:
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: Warning: no collective variables defined.
colvars: Collective variables initialized, 0 in total.
colvars: No collective variables biases were defined.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1 [default]
colvars: # colvarsRestartFrequency = 10 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rgyr"
colvars:   Initializing a new "gyration" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { HA }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 15 atoms requested.
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1 [default]
colvars:   # hardLowerBoundary = on [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = on
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rgyr }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 6 }
colvars:   # targetCenters = { 6 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 1
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "rgyr" will be rescaled to 1 according to the specified width (1).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (15 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - Colvars module:
colvars: - Colvars-NAMD interface:
colvars: - Harmonic colvar bias implementation:
colvars: - Optimal rotation via flexible fitting:
colvars: - gyration colvar component:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - NAMD engine:
colvars:   Phillips2020 https://doi.org/10.1063/5.0014475
colvars: 
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Resetting the Collective Variables module.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration:
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: Warning: no collective variables defined.
colvars: Collective variables initialized, 0 in total.
colvars: No collective variables biases were defined.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1 [default]
colvars: # colvarsRestartFrequency = 10 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rgyr"
colvars:   Initializing a new "gyration" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { HA }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 15 atoms requested.
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1 [default]
colvars:   # hardLowerBoundary = on [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = on
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rgyr }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 6 }
colvars:   # targetCenters = { 6 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 1
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "rgyr" will be rescaled to 1 according to the specified width (1).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (15 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
